### ISSUE TITLE
Rename maven-osgi-exec-plugin to equinox-maven-plugin

### ISSUE THREAD

**Issue by kenwenzel** [BestSolution-at/maven-osgi-plugin#30]
maven-osgi-exec-plugin does currently not follow Maven's naming conventions for 3rd party plugins:

```
Artifact Ids of the format maven-___-plugin are reserved for 
plugins in the Group Id org.apache.maven.plugins
Please change your artifactId to the format ___-maven-plugin
In the future this error will break the build.
```
I would suggest to simply name the plugin `equinox-maven-plugin`.
Later it may provide the goals `configure`, `run` (instead of exec-osgi) and `test`.

**Comment by kenwenzel** [comment]
Maybe it is also a good idea to move this plugin into a project on its own.

**Comment by kenwenzel** [comment]
Do you have any suggestions on the next step for this issue?

**Comment by tomsontom** [comment]
I'd like to keep this all in one github project. 

On the name what if we name it "equinox-*-maven-*"?
We'd then have:
* equinox-launch-maven-plugin
* equinox-package-maven-plugin
* equinox-targetplatform-maven-extension
* equinox-targetplatform-maven-lib
* equinox-targetplatform-maven-plugin

**Comment by kenwenzel** [comment]
Yes, that is also ok - although the package and targetplatform plugins are more or less p2/pde specific.

**Comment by kenwenzel** [comment]
Maybe equinox-launch-maven-plugin could also be named equinox-run-maven-plugin (like bnd-run-maven-plugin)...

**Comment by kenwenzel** [comment]
Dear @tomsontom, 
can I do anything to accelerate the refactoring?
Best regards, Ken

**Comment by tomsontom** [comment]
I'm swapped with work but i'll try to reserve some time next week to work on that

**Comment by tomsontom** [comment]
gosh sorry - this completely fell of my list

**Comment by kenwenzel** [comment]
Please let me know if I can do something :-)
